language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -119,7 +123,7 @@ instance ExMemoryUsage () where
   memoryUsage _ = 0 -- TODO or 1?
 
 instance ExMemoryUsage Integer where
-  memoryUsage _ = 2 -- TODO
+  memoryUsage i = ExMemory (if i == 0 then 0 else smallInteger (integerLog2# (abs i) `quotInt#` (integerToInt 60))) -- assume 60bit size
```

Comment:
60 bits is rather unexpected here.  What's going on?

Reply:
I assume 60bits in an Integer, whereas you probably get 64bit per word. So it's a bit of an underestimation. Might even be +1 for all the bookkeeping information.

Reply:
Sounds like it needs a comment :)